app/webpacker/components/Panel/pages/RunValidatorsPage/ValidationListView.jsx

```diff
@@ -0,0 +1,56 @@
+import React, { Fragment, useMemo } from 'react';
+import _ from 'lodash';
+import { Header, List, ListItem } from 'semantic-ui-react';
+import I18n from '../../../../lib/i18n';
+import { competitionUrl } from '../../../../lib/requests/routes.js.erb';
+
+const headingPrefixForType = (type) => {
+  switch (type) {
+    case 'info': return 'Information for';
+    case 'warning': return 'Warnings detected in';
+    case 'error': return 'Errors detected in';
+    default: return 'Unknown detected in';
+  }
+};
+
+export default function ValidationListView({ validations, showCompetitionNameOnOutput, type }) {
+  const listByGroup = useMemo(() => _.groupBy(validations, 'kind'), [validations]);
+
+  return (
+    <>
+      {Object.entries(listByGroup).map(([group, list]) => (
+        <Fragment key={group}>
```

Comment:
Instead of `Fragment`, you could consider wrapping this in a `Message`. SemUI Messages have `info`, `warning` and `error` states with nice color codings :)

Reply:
Whatever works for you! I won't block this PR only because of this design choice.
But I do want to point out that you can have a list inside the body of a `Message`: https://react.semantic-ui.com/collections/message/#types-list

Reply:
Oh you meant to have one `Message` for one section right? That actually makes sense. I thought each point should have been a message, which I felt a bit as an overkill. I've made the change. Thank you.